draft-enghardt-panrg-path-properties.md

```diff
@@ -66,63 +64,62 @@ in Path-Aware Networking {{I-D.irtf-panrg-questions}}, which is a product of the
-An endpoint can for example choose its access provider, but cannot choose the backbone path to a given destination since the access provider will make their own policy-based routing decision.
+Typically, backbone properties are less easily available to an endpoint than domain properties, due to the potential increased distance and the lack of pre-existing trust or contractual relationship.
 
-Presence of certain device on the path:
-: Could be the presence of a certain kind of middlebox, e.g., a proxy, a firewall, a NAT.
+Additionally, endpoints are less likely to be able to influence which path elements form their path in the backbone, as well as their properties.
 
-Presence of a packet forwarding node or specific Autonomous System on a path:
-: Indicates that traffic goes through a certain node or AS, which might be relevant for deciding the level of trust this path provides.
+Some path properties relate to the entire path, part of which often lies outside of an endpoint's administrative domain. Thus, such properties are listed as Backbone Properties.
+
+
+Administrative Entity:
+: The administrative entity, e.g., the AS, to which a path element or set of path elements belongs.
 
 Disjointness:
-: How disjoint a path is from another path.
+: For a set of two paths, the set of path elements which are shared.
 
 Path MTU:
 : The maximum size, in octets, of an IP packet that can be transmitted without fragmentation on a path segment or path.
 
 Transport Protocols available:
-: Whether a specific transport protocol can be used to establish a connection over this path. An endpoint may know this because it has cached whether it could successfully establish, e.g., a QUIC connection, or an MPTCP subflow.
+: Whether a specific transport protocol can be used to establish a connection over a path or path segment (ordered set of path elements). An endpoint may cache its knowledge about recent successfully established connections using specific protocols, e.g., a QUIC connection, or an MPTCP subflow, over a specific path.
 
 Protocol Features available:
-: Whether a specific feature within a protocol is known to work over this path, e.g., ECN, or TCP Fast Open.
+: Whether a specific protocol feature is available over this path, e.g., Explicit Congestion Notification (ECN), or TCP Fast Open.
 
 
 # Dynamic Properties
 
-Dynamic Path Properties are expected to change on the timescale of milliseconds.
-They usually relate to the state of the path, such as the current end-to-end latency.
-Some of these properties may depend only on the first hop or on the access network, some may depend on the entire path.
-Properties related to a single layer 2 domain are abstracted from the used physical and link layer technology, similar to {{RFC8175}}.
+Dynamic path properties relate to a path or path element with respect to the transmission of an individual packet or of a flow over a path element or path.
```

Comment:
I would use the previously defined path segment ;)
"Dynamic path properties relate to a path segment with respect to the transmission of an individual packet or flow over this path segment."

Maybe we should define path segment in the terminology section?

Reply:
Yes, we definitely should!

As the terminology section is merged now, I'm merging it into this branch and then adding the missing definitions.

Reply:
Oh, and I just realized that as is, a path segment is the same as a path. But a path should be defined end-to-end, right? So I'm adding "between two endpoints" to the path definition.

Reply:
Yes that sounds good